participation.html

```diff
@@ -13,7 +13,13 @@
     <div class="banner white">
       <div class="container">
         <h3 class="size8">Scout/Parent Participation</h3>
-        <p>Scout and parent participation are crucial to the success of Troop 370.  Learning, leadership, and advancement opportunities abound for Scouts.<br><br>For parents, there are many ways in which you can assist through volunteer efforts.  There is no greater joy than joining your Scout through their Troop 370 experience, and working to support the Troop "behind the scenes".  If you would like to see how you can help, please contact the Troop, attend a meeting, or check out the "What Can I Do?" link in the password protected members section of the site.  Meetings are held most Wednesday evenings from 7:30 - 9:00PM in the St. James UMC activities building.<br><br>Using the drop-down menu under "Scout/Parent Info" above, you will find forms, helpful information, and other common documents that you will find helpful during your journey through Scouts at Troop 370.</p>
+        <p>Scout and parent participation are crucial to the success of Troop 370.  Learning, leadership, and advancement opportunities abound for Scouts.</p>
+        <p>At Troop 370, there are monthly weekend campouts that the troop takes. Also, every summer, the Troop visits Woodruff Scout Reservation in Blairsville, GA. Recently, they have also been visiting the Summit Bechtel Reserve in West Virginia as a secondary summer camp. The Troop has a multitude of high adventure trips as well. They visit all high adventure bases (more info on high adventure <href> here). The troop also has many service projects open to scout participation.</p>
```

Comment:
```suggestion
        <p>Additionally, Troop 370 has weekly meetings, goes on monthly campouts, attends multiple summer camps, leads dozens of Eagle Scout service projects, and enjoys multiple high adventure opportunities throughout the year. We encourage every youth scout and adult scouter to participate in these wonderful opportunities provided by the Troop. Learn more about our campouts and other events at the <a href="camps-adventure.html">Camps/High Adventure</a>and <a href="campout-schedule-local-events.html">Campout Schedule</a>pages</p>
```
These changes make this paragraph a more specific overview of what the Troop does while pointing the reader to the pages with the exact events.

I think it makes more sense to move this paragraph to after the paragraph about advancement. This provides a nice link between the scout participation and adult participation paragraphs because the troop events are events for both scouts and scouters.